experiments/scripts/run_tune.py

```diff
@@ -152,9 +159,12 @@ def run(config_path: str, config_overrides: Optional[Dict] = None) -> None:
 
     else:
         # load single dataset
-        train_dataset, val_dataset = load_and_split(
-            config.data.path, config.data.validation_size
-        )
+        loaded_dataset = load_and_split(config.data.path, config.data.validation_size)
+        train_dataset, val_dataset = loaded_dataset["train"], loaded_dataset["test"]
```

Comment:
Is it really the `test` split or should it be the `valid` split?

Reply:
Ah, got it, afaik it will only split (random) when there is no test subset there?
We should discuss on how to best set up a test subset that we will use for all future runs.